package.json

```diff
@@ -146,7 +146,7 @@
 		"storybook-dark-mode": "^1.0.3",
 		"ts-node": "^9.1.1",
 		"tslib": "^2.0.3",
-		"typescript": "^3.9.7"
+		"typescript": "^4.2.4"
```

Comment:
Is this version update needed?

Reply:
@WoodNeck 

ResizeObserver is supported from typescript version 4 or later.